tests/import_export_test.py

```diff
@@ -96,22 +97,15 @@ def test_import_parquet(self):
         rmtree(f_base)
 
     def test_export_parquet(self):
-        f_base = f"{os.getcwd()}/import_export_test"
-        # make directory to save to so pandas read works
-        if not os.path.exists(f_base):
-            os.mkdir(f_base)
-
         akdf = self.build_arkouda_dataframe()
-        akdf.save(f"{f_base}/ak_write", file_format="Parquet")
-        print(akdf.__repr__())
+        with tempfile.TemporaryDirectory(dir=ImportExportTest.ie_test_base_tmp) as tmp_dirname:
+            akdf.to_parquet(f"{tmp_dirname}/ak_write")
+            print(akdf.__repr__())
 
-        pddf = ak.export(f"{f_base}/ak_write", write_file=f"{f_base}/pd_from_ak.parquet", index=True)
-        print(pddf)
-        self.assertEqual(len(glob.glob(f"{f_base}/pd_from_ak.parquet")), 1)
-        self.assertTrue(pddf.equals(akdf.to_pandas()))
-
-        with self.assertRaises(RuntimeError):
-            pddf = ak.export(f"{f_base}/foo.h5", write_file=f"{f_base}/pd_from_ak.h5", index=True)
+            pddf = ak.export(f"{tmp_dirname}/ak_write", write_file=f"{tmp_dirname}/pd_from_ak.parquet", index=True)
+            print(pddf)
```

Comment:
I know this is not new code but I want to make sure we should be printing in this test

Reply:
I don't think we need to. I updated this.